workers/unity/Packages/io.improbable.gdk.gameobjectcreation/AssemblyInfo.cs

```diff
@@ -7,4 +7,4 @@
 [assembly: InternalsVisibleTo("Improbable.Gdk.EditmodeTests")]
 [assembly: InternalsVisibleTo("Improbable.Gdk.PlaymodeTests")]
 
-[assembly: RegisterGenericComponentType(typeof(GameObjectInitializationComponent))]
+[assembly: RegisterGenericComponentType(typeof(GameObjectInitSystemStateComponent))]
```

Comment:
Is this needed? Its not a generic component

Reply:
strange, it used to throw errors about this before but not anymore!

Reply:
(I'm removing it)